clang/test/SemaSYCL/spec-const-kernel-arg.cpp

```diff
@@ -24,10 +24,10 @@ int main() {
   });
 }
 
-// CHECK: FunctionDecl {{.*}}kernel_sc{{.*}} 'void ()'
-// CHECK: VarDecl {{.*}}'(lambda at {{.*}}'
-// CHECK-NEXT: InitListExpr {{.*}}'(lambda at {{.*}}'
+// CHECK: FunctionDecl {{.*}}kernel_sc{{.*}} 'void (sycl::ONEAPI::experimental::spec_constant<char, class MyInt32Const>, sycl::ONEAPI::experimental::spec_constant<int, class sc_name1>, sycl::ONEAPI::experimental::spec_constant<int, class sc_name2>)'
```

Comment:
I don't think this test should change, doesn't seemed affected by the new attribute.